CMSIS/Core/Include/pmu_armv8.h

```diff
@@ -31,6 +31,1036 @@
 #ifndef ARM_PMU_ARMV8_H
 #define ARM_PMU_ARMV8_H
 
+/** \brief PMU Events */
 
-#endif /* ARM_PMU_ARMV8_H */
+#define ARM_PMU_SW_INCR                              0x0000
```

Comment:
We should add descriptive documentation to all these events. What does this event "count"?

Reply:
Done.